drivers/temperature/adt7420/adt7420.h

```diff
@@ -45,74 +45,76 @@
+#define ADT7320_REG_T_LOW    	    0x07 // Temperature LOW setpoint (10'C)
 
 /* ADT7320 SPI command byte */
-#define ADT7320_WRITE_MASK_CMD          0b00111000 // SPI write command
-#define ADT7320_READ_CMD                0b01000000 // SPI read command
+#define ADT7320_WRITE_MASK_CMD      0b00111000 // SPI write command
+#define ADT7320_READ_CMD            0b01000000 // SPI read command
 
 /* ADT7420 (I2C) registers */
-#define ADT7420_REG_TEMP_MSB		    0x00 // Temperature value MSB
-#define ADT7420_REG_TEMP_LSB		    0x01 // Temperature value LSB
-#define ADT7420_REG_STATUS			    0x02 // Status
-#define ADT7420_REG_CONFIG			    0x03 // Configuration
-#define ADT7420_REG_T_HIGH_MSB		    0x04 // Temperature HIGH setpoint MSB
-#define ADT7420_REG_T_HIGH_LSB		    0x05 // Temperature HIGH setpoint LSB
-#define ADT7420_REG_T_LOW_MSB		    0x06 // Temperature LOW setpoint MSB
-#define ADT7420_REG_T_LOW_LSB		    0x07 // Temperature LOW setpoint LSB
-#define ADT7420_REG_T_CRIT_MSB		    0x08 // Temperature CRIT setpoint MSB
-#define ADT7420_REG_T_CRIT_LSB		    0x09 // Temperature CRIT setpoint LSB
-#define ADT7420_REG_HIST			    0x0A // Temperature HYST setpoint
-#define ADT7420_REG_ID				    0x0B // ID
-#define ADT7420_REG_RESET			    0x2F // Software reset
+#define ADT7420_REG_TEMP_MSB		0x00 // Temperature value MSB
+#define ADT7420_REG_TEMP_LSB		0x01 // Temperature value LSB
+#define ADT7420_REG_STATUS		0x02 // Status
+#define ADT7420_REG_CONFIG		0x03 // Configuration
+#define ADT7420_REG_T_HIGH_MSB		0x04 // Temperature HIGH setpoint MSB
+#define ADT7420_REG_T_HIGH_LSB		0x05 // Temperature HIGH setpoint LSB
+#define ADT7420_REG_T_LOW_MSB		0x06 // Temperature LOW setpoint MSB
+#define ADT7420_REG_T_LOW_LSB		0x07 // Temperature LOW setpoint LSB
+#define ADT7420_REG_T_CRIT_MSB		0x08 // Temperature CRIT setpoint MSB
+#define ADT7420_REG_T_CRIT_LSB		0x09 // Temperature CRIT setpoint LSB
+#define ADT7420_REG_HIST		0x0A // Temperature HYST setpoint
+#define ADT7420_REG_ID			0x0B // ID
+#define ADT7420_REG_RESET		0x2F // Software reset
 
 /* ADT7420_REG_STATUS definition */
-#define ADT7420_STATUS_T_LOW		    (1 << 4)
-#define ADT7420_STATUS_T_HIGH		    (1 << 5)
-#define ADT7420_STATUS_T_CRIT		    (1 << 6)
-#define ADT7420_STATUS_RDY			    (1 << 7)
+#define ADT7420_STATUS_T_LOW		BIT(4)
+#define ADT7420_STATUS_T_HIGH		BIT(5)
+#define ADT7420_STATUS_T_CRIT		BIT(6)
+#define ADT7420_STATUS_RDY		BIT(7)
 
 /* ADT7420_REG_CONFIG definition */
 #define ADT7420_CONFIG_FAULT_QUEUE(x)	(x & 0x3)
-#define ADT7420_CONFIG_CT_POL		    (1 << 2)
-#define ADT7420_CONFIG_INT_POL		    (1 << 3)
-#define ADT7420_CONFIG_INT_CT_MODE  	(1 << 4)
-#define ADT7420_CONFIG_OP_MODE(x)	    ((x & 0x3) << 5)
-#define ADT7420_CONFIG_RESOLUTION	    (1 << 7)
+#define ADT7420_CONFIG_CT_POL		BIT(2)
+#define ADT7420_CONFIG_INT_POL		BIT(3)
+#define ADT7420_CONFIG_INT_CT_MODE	BIT(4)
+#define ADT7420_CONFIG_OP_MODE(x)	(x << 5) & (GENMASK(6,5))
```

Comment:
Always put parantheses around macro parameter when referencing it in the expression, this should be:
`((x) << 5) ...`

If you don't put parantheses it may not work like you intend.
Without parantheses, `ADT7420_CONFIG_OP_MODE(a + b)` would work because it translates as `(a + b << 5) ...` and + is more prioritary than <<. But  `ADT7420_CONFIG_OP_MODE(a & b)` would not work because & is less prioritary than << in C operator precedence. https://en.cppreference.com/w/c/language/operator_precedence

Reply:
Thanks for flagging this, added parenthesis, considering the above mentioned case.